salt/salt.spec

```diff
@@ -330,8 +330,8 @@ Requires:       iproute
 %endif
 
 %if %{with systemd}
-BuildRequires:  pkgconfig(systemd)
```

Comment:
Same question as for the other PR

Reply:
See comment on other PR I didn't make that change. It was already there in the OBS branch I used. https://build.opensuse.org/package/view_file/systemsmanagement:Uyuni:Master:CentOS8-Uyuni-Client-Tools/salt/salt.spec?expand=1